app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -40,6 +39,7 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
     private var isSearchOpened = false
     private var edtSearch: EditText? = null
     private var skills: ArrayList<Pair<String, List<SkillData>>> = ArrayList()
+    var text: String = ""
```

Comment:
make this local , no need for this to be global

Reply:
@batbrain7 I made it earlier, I was asked to make it global!

Reply:
@codedsun Why not a private var?

Reply:
@arundhati24 I was asked to make it global earlier as far as I remember!

Reply:
This can still be private, global doesn't mean public